onnx_tf/handlers/frontend/zeros_like.py

```diff
@@ -1,14 +1,36 @@
+import numpy as np
+
+from onnx_tf.common import get_unique_suffix
 from onnx_tf.handlers.frontend_handler import FrontendHandler
+from onnx_tf.handlers.frontend.shape import Shape
 from onnx_tf.handlers.handler import onnx_op
 from onnx_tf.handlers.handler import tf_op
+from onnx_tf.pb_wrapper import TensorflowNode
 
 
-@onnx_op("Constant")
+@onnx_op("ConstantOfShape")
 @tf_op("ZerosLike")
```

Comment:
This handler is not getting triggered since tf.zeros_like python api produces a node with op_type='const' or 'fill', according to https://github.com/tensorflow/tensorflow/blob/6612da89516247503f03ef76e974b51a434fb52e/tensorflow/python/ops/array_ops.py#L1803. The similar message is in tensorflow release.md, 'ZerosLike and OnesLike ops treated as constants by Graph Transform Tool.'

Reply:
We should remove the frontend handler for zero_like, but in our support document need to tell user zero_like is supported.

Reply:
@chinhuang007 the link you gave point to the implementation of tf.zeros.

Reply:
@tjingrant zeros_like call zeros_like_impl and zeros_like_impl call zeros

Reply:
> ZerosLike and OnesLike ops treated as constants by Graph Transform Tool.

I fail to see how this pertain to our discussion. ZerosLike may appear to be constant for graph transform tool, but for us it is clearly not constant. Just think about tf.zeros_like whose input is a batch of images with dynamically determined batch size.

Reply:
@winnietsang the path you pointed out is existential not universal. I can send you an email with a model with ZeroLike ops exported with tf 1.13.1 .

Reply:
@tjingrant The fact that the tf.zeros_like python api produces a node with op_type='const' or 'fill' is relevant in our test case as minimum. Please note test_cases.append(("test_zeros_like", **tf.zeros_like,** "zeros_like", [get_rnd([5, 5])], {})) will never trigger our handler, for the obvious reason, the op_type is NOT "ZerosLike"!

Reply:
@chinhuang007 I agree the test case is insufficient to trigger this handler. But this is a problem of our testing methodology in this particular context, not a problem for the implementation of this handler. And this handler certainly gets triggered outside our tests on real models.

Reply:
@tjingrant Please send a model with ZerosLike ops exported with tf 1.13.1. We will need to find a way to write test cases for this kind of ops that do not have a matching python api. Otherwise we would never know for sure whether the handler works or not.

Reply:
@chinhuang007 already did : )

Reply:
@tjingrant Just saw it :-) It would become part the graph because it is a legit C++ op. Honestly I am very interested in how it got into the graph. Regardless, the question remains, how do we test and verify our frontend handlers for these ops with no python APIs?